OpenEdXMobile/src/main/java/org/edx/mobile/view/AccountFragment.kt

```diff
@@ -62,6 +63,8 @@ class AccountFragment : BaseFragment() {
         }
         binding.tvVersionNo.text = String.format("%s %s %s", getString(R.string.label_version),
                 BuildConfig.VERSION_NAME, environment?.config?.environmentDisplayName)
+
+        environment?.analyticsRegistry?.trackScreenView(Analytics.Screens.ACCOUNT_SETTINGS)
```

Comment:
better to move in the `onViewCreated` method (if possible). cuz trigger event when the screen is completely visible.

Reply:
done